app/views/layouts/application.html.erb

```diff
@@ -27,7 +27,9 @@
     <%= yield %>
   </main>
 
-  <p class="govuk-body-s">Version: <%= CURRENT_RELEASE_SHA %></p>
+  <% if !CURRENT_RELEASE_TAG.nil? %>
+    <p class="govuk-body-s">Version: <%= CURRENT_RELEASE_TAG %></p>
+  <% end %>
```

Comment:
At this point I'd be inclined to remove the initializer and just do something like:

```suggestion
  <% if ENV.key? "SENTRY_RELEASE" %>
    <p class="govuk-body-s">Version: <%= ENV["SENTRY_RELEASE"] %></p>
  <% end %>
```

Up to you though, I'm happy either way